build/util/src/lib.rs

```diff
@@ -82,6 +84,44 @@ pub fn task_maybe_config<T: DeserializeOwned>() -> Result<Option<T>> {
     toml_from_env("HUBRIS_TASK_CONFIG")
 }
 
+pub fn task_peripherals() -> BTreeMap<String, Peripheral> {
+    ron::de::from_str(&env::var("HUBRIS_TASK_PERIPHERALS").unwrap()).unwrap()
```

Comment:
Can we get a more verbose error message for this.  The unwraps are `env::var` are really hard to debug if they fail.  In my experience it just throws a generic `environment variable not found` and even `RUST_BACKTRACE=1` doesn't help.

Reply:
Done in next revision